.github/workflows/main-ci.yml

```diff
@@ -2,24 +2,24 @@ name: CI - Main
 
 on:
   push:
-     branches:
-       - '*'
-       - '!skip-ci-*'
-       - '!DEFEDIT-*'
-       - '!editor-dev'
+    branches:
+      - 'hotfix-*'
+      - 'dev'
+      - 'beta'
+      - 'master'
```

Comment:
`editor-dev` is excluded here since it is in `editor-only.yml`. Note that the indentation was inconsistent in this file. I think it still worked, though.